cea/technologies/boilers.py

```diff
@@ -104,18 +104,18 @@ def cond_boiler_op_cost(Q_therm_W, Q_design_W, T_return_to_boiler_K, BoilerFuelT
 
 
     if BoilerFuelType == 'BG':
```

Comment:
'BG' should probably be written as a constant, like, `BIO_GAS`? (i'm assuming that's what it stands for?)

And since you went to the trouble of creating a class for prices, this would be an *excellent* place to refactor the `if` to a method: `gas_price = prices.gas_price(boiler_fuel_type)`

Reply:
(apply that to the stuff below too)